go.mod

```diff
@@ -1,9 +1,12 @@
-module github.com/VoltDB/voltdb-client-go
+module voltdb-client-go
```

Comment:
I don't understand the use of this file so cannot review

Reply:
Will revert all changes to these